includes/admin/customers/class-customer-addresses-table.php

```diff
@@ -210,18 +210,21 @@ public function column_customer( $item ) {
 	 * @access public
 	 * @since 3.0
 	 *
-	 * @param array Address
+	 * @param array $item Address object.
 	 *
 	 * @return string Displays a checkbox
 	 */
 	public function column_cb( $item ) {
 		$customer = edd_get_customer_by( 'id', $item['customer_id'] );
-		$name     = isset( $customer->name ) ? $customer->name : $customer->email;
+		$name     = sprintf( __( 'Address ID: ' ) . $item['id'] );
```

Comment:
The text string is missing the placeholder to insert the ID (you won't concatenate with the `.`), and is missing the text domain.